Buildings/Examples/VAVReheat/Validation/BaseClasses/Floor.mo

```diff
@@ -61,8 +66,8 @@ Buildings.Examples.VAVReheat.Validation.TraceSubstance</a>.  It extends
 <a href=\"modelica://Buildings.Examples.VAVReheat.BaseClasses.Floor\">
 Buildings.Examples.VAVReheat.BaseClasses.Floor</a> with CO<sub>2</sub> generation
 from people and CO<sub>2</sub> from outside air infiltration.  Calculation of the
-generation from people uses the gain schedule for latent load and assumes
-80 W per person latent load to calculate the number of people.  Then, it assumes
+generation from people uses the internal gain schedule and uses 0.05 ppl/m^2
```

Comment:
@dhblum : do you mean PPM/m<sup>2</sup> during occupied hours rather than ppl?

Reply:
No, 0.05 is the assumed design occupancy density in the space so it should be ppl (or people).  PPM per person is discussed in the next sentence.

Reply:
Ah, now I understand what ppl means. I made a commit that spells out the abbreviation.

Reply:
Sounds good thanks 👍